README.md

```diff
@@ -233,7 +233,9 @@ config for the OpenShift Container Platform. WMCO will not be able to automatica
 workloads.
 
 ### Storage
-At this time, only in-tree storage is supported in all cloud providers.
+Windows Nodes are running csi-proxy and are ready to use CSI drivers, however Windows CSI driver DaemonSets are not
+managed by the cluster. In order to use persistent storage for Windows workloads, the cluster administrator should
+follow upstream documentation to deploy the Windows CSI driver DaemonSet for the desired storage provider.
```

Comment:
```suggestion
follow upstream or cloud provider documentation to deploy the Windows CSI driver DaemonSet for the desired storage provider.
```

Reply:
Reworded, but not all storage drivers == cloud provider. For example SMB